src/app/QuickStarts/quickstarts/dashboard-quickstart.tsx

```diff
@@ -58,127 +59,124 @@ const DashboardQuickStart: QuickStart = {
+Dashboard Cards are displayed in **Dashboard Layouts**, which can be customized by adding, removing, or rearranging cards. The layout can also be saved and restored at a later time. 
 
 ### What you'll learn
 - How to create a simple Dashboard Layout
 - How to add and remove a Dashboard Card to/from a Dashboard Layout
 - How to move and resize Dashboard Cards
 - How to rename, upload, download, and delete Dashboard Layouts
 
-[Learn more about each Dashboard Card in the [Cryostat documentation](${build.dashboardGuideUrl}).]{{admonition tip}}
     `,
     tasks: [
       {
         title: 'Go to the Dashboard page',
         description: `
-1. Click the [Dashboard]{{highlight nav-dashboard-tab}} tab in the [APP] console navigation bar.
-2. Select a target JVM from the [Target Selector]{{highlight target-select}} on the toolbar.
+1. In the [APP] console navigation bar, click [Dashboard]{{highlight nav-dashboard-tab}}.
+2. From the [Target Selector]{{highlight target-select}}, select a target JVM.
 `,
         review: {
           instructions: '#### Verify that you see the Dashboard page.',
           failedTaskHelp:
-            'If you do not see the navigation bar, you can click the `☰` button in the [top left corner of the page]{{highlight nav-toggle-btn}}.',
+            'If you do not see the navigation bar, click the [menu button]{{highlight nav-toggle-btn}} on the masthead.',
         },
       },
       {
         title: 'Create a new Dashboard Layout',
         description: `
-**Dashboard Layouts** are used to organize the Dashboard Cards that are displayed in the Dashboard. We will start by creating a new Dashboard Layout.
+**Dashboard Layouts** are used to organize the Dashboard Cards that are displayed in the Dashboard.
 
 1. Click the [Layout Selector]{{highlight dashboard-layout-selector}} on the toolbar.
 
     This will open a dropdown menu. Click **New Layout**.
 
-We have now created a new Dashboard Layout. The new layout is automatically selected and should be named **Custom1**, if you have not created any other layouts yet.
+The new layout is automatically selected and should be named **Custom1**, assuming you have not created any other layouts.
 
 [You can also rename the layout with the pencil icon next to the name.]{{admonition tip}}
 `,
         review: {
-          instructions:
-            '#### Verify that the new layout is created and selected. Make sure that the the new layout is named Custom with some number appended to it.',
+          instructions: '#### Verify that the new layout is created and selected.',
           failedTaskHelp:
-            'After opening the [Layout Selector]{{highlight dashboard-layout-selector}} dropdown, click the blue button labeled **New Layout**.',
+            'Try the steps again. Make sure that the the new layout is named Custom with some number appended to it.',
```

Comment:
-> `with a numeric suffix`.

This way it does not raise the question about appending where (i.e. front? or end?)